test/rails_app/config/initializers/assets.rb

```diff
@@ -0,0 +1 @@
+Rails.application.config.assets.version = '1.0'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.